Website/src/rendering/src/components/Sponsors/SelectedSponsorsGrid.tsx

```diff
@@ -10,6 +15,13 @@ export type SelectedSponsorsGridProps = ComponentProps & {
 };
 
 const SelectedSponsorsGrid = (props: SelectedSponsorsGridProps): JSX.Element => {
+  const { sitecoreContext } = useSitecoreContext<SitecoreContextValue>();
+
+  const isPageEditing = sitecoreContext.pageState === LayoutServicePageState.Edit;
+  const hasSponsors = props?.fields?.Sponsors.length;
```

Comment:
`hasSponsors` is an integer or `undefined` with that code. It could even throw if `Sponsors` is undefined. We could use `!!props?.fields?.Sponsors?.length` for it to always be a boolean.